requirements/main.in

```diff
@@ -58,7 +58,7 @@ structlog
 transaction
 trove-classifiers
 typeguard
-webauthn<1.0.0  # https://github.com/pypa/warehouse/pull/10202
+webauthn>=1.0.0,<2.0.0
```

Comment:
If I'm not mistaken, this can be simply written as

```suggestion
webauthn=1.*
```

Reply:
I've never seen or used that, will stick with the current specifier for consistency with the other statements in the file.